python/requirements.txt

```diff
@@ -9,3 +9,6 @@ scout.py==0.5.0
 semantic-version==2.6.0
 kubernetes==8.0.0
 click==7.0
+pytest
```

Comment:
Note that `pytest` and `pyyaml` are _development_ dependencies, but they should not be _runtime_ dependencies, because we shouldn't be shipping `kat` in the Docker image at all. So they should really be in `dev-requirements.txt` instead of here.

Reply:
Done.